benchmark_runner/common/oc/oc.py

```diff
@@ -57,11 +57,25 @@ def get_kata_version(self):
         """
         return self.run("oc get csv -n openshift-sandboxed-containers-operator $(oc get csv -n openshift-sandboxed-containers-operator --no-headers | awk '{ print $1; }') -ojsonpath='{.spec.version}' ")
 
+    def _get_kata_default_channel(self):
```

Comment:
Line 70 of oc.py:

        default_channel = f'"{self.**_get_kata_default_channel()**}"'

The reason for the odd construct is so that the double quotes are part of the string.